src/ObjectMultiplex.ts

```diff
@@ -25,12 +25,12 @@ export class ObjectMultiplex extends Duplex {
   createStream(name: string): Substream {
     // guard stream against destroyed already
     if (this.destroyed) {
-      throw new Error('ObjectMultiplex - parent stream already destroyed');
+      throw new Error(`ObjectMultiplex - parent stream for name "${name}" already destroyed `);
```

Comment:
Nit: Extra space here

```suggestion
      throw new Error(`ObjectMultiplex - parent stream for name "${name}" already destroyed`);
```